src/semantic_csv/impl/core.cljc

```diff
@@ -74,13 +75,14 @@
 (defn- subset-map [m ks]
   (into {} (map (fn [k] [k (get m k)]) ks)))
 
-(defmacro clone
```

Comment:
It works for cljs, it's just that macros have to be defined in clojure, then imported to clojurescript. In the ns declaration I put a refer-macros form, which makes the file self referential. This allows for macros to be pulled in by refer in cljs land